srcpkgs/mercurial/template

```diff
@@ -1,23 +1,39 @@
 # Template file for 'mercurial'
 pkgname=mercurial
-version=5.6.1
+version=5.8
 revision=1
 build_style=python3-module
-hostmakedepends="python3 python3-setuptools python3-devel"
+hostmakedepends="python3 python3-setuptools python3-devel gettext"
 makedepends="python3-devel"
 depends="python3 ca-certificates"
 checkdepends="iana-etc tar unzip which xz"
+# cvs tests fail
+checkdepends="iana-etc tar unzip which xz subversion-python python3-docutils gnupg2
```

Comment:
don't duplicate the variable

Reply:
Done.

Reply:
Pushed again - removed the wrong line... :(